src/views/ApplicationViews.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react"
+import { Outlet, Route, Routes } from "react-router-dom"
+import { Categories } from "../components/categories/Categories";
+import { Welcome } from "../components/welcome/Welcome";
+
+export const ApplicationViews = () => {
+    const [currentUser, setCurrentUser] = useState({});
```

Comment:
I would suggest removing this unused useState line until the authentication step is implemented in a future pull request to avoid any "ghost code" popping up in the future.